.release-version

```diff
@@ -0,0 +1 @@
+3.12.1
```

Comment:
This should be incremented.  It would normally pick this up as part of the PR, except the current version 3.12.1 isn't consistently tagged.